seatunnel-e2e/seatunnel-connector-v2-e2e/connector-jdbc-e2e/connector-jdbc-e2e-part-2/src/test/java/org/apache/seatunnel/connectors/seatunnel/jdbc/JdbcOceanBaseMilvusIT.java

```diff
@@ -281,6 +286,63 @@ public void testFakeToOceanBase(TestContainer container)
         }
     }
 
+    @TestTemplate
+    public void testOceanBaseToMilvus(TestContainer container) throws Exception {
+        try {
+            initOceanBaseTestData();
+            Container.ExecResult execResult = container.executeJob(configFile().get(2));
```

Comment:
```suggestion
            Container.ExecResult execResult = container.executeJob("/jdbc_oceanbase_source_and_milvus_sink.conf");
```
This way is more stable